### PR TITLE
did class_five assgnmt and class_six class work

### PR DESCRIPTION
**Pull request by BuchiGold**
assignment and classwork

### CODE REVIEW

**Comment by BuchiGold** [comment]
assignmt and classwork

File: class_six/food1.avif
Comment (Sesughter01) [inline class_six/food1.avif:1]: please use any of the following file formats: png, jpg for your images

Also move this file to the assets folder in your root directory

File: class_six/food2.avif
Comment (Sesughter01) [inline class_six/food2.avif:1]: same thing for this one

File: class_six/food3.avif
Comment (Sesughter01) [inline class_six/food3.avif:1]: same thing here

File: class_six/football.avif
Comment (Sesughter01) [inline class_six/football.avif:1]: same thing here too

File: class_six/github.jpg
Comment (Sesughter01) [inline class_six/github.jpg:1]: This is a jpg file this is ok but move it into the assets folder

File: class_six/index.html
Comment (Sesughter01) [inline class_six/index.html:1]: buchi_corrctns
Your item1 is wrongly positioned and i said to replace the header text with your logo either as text logo or an actual logo.

File: class_six/twitter.jpg
Comment (Sesughter01) [inline class_six/twitter.jpg:1]: move to asset folder

File: class_six/vercel.jpg
Comment (Sesughter01) [inline class_six/vercel.jpg:1]: move to assets folder

**Review by Sesughter01 — CHANGES_REQUESTED** [review]
Please take a very close look at the corrections made and make changes

**Comment by BuchiGold** [comment]
made necessary corrections

**Pull request closed without merging**